py/objint_longlong.c

```diff
@@ -40,14 +40,6 @@
 
 #if MICROPY_LONGINT_IMPL == MICROPY_LONGINT_IMPL_LONGLONG
 
-// Python3 no longer has "l" suffix for long ints. We allow to use it
```

Comment:
I just saw a definition which was last used in a function which got removed and it wasn't clear to me from the comment that the point of SUFFIX is to arbitrarily use it when needed for debugging.
Anyway: proper fix is then to also introduce e.g. MICROPY_DEBUG_LONGINT ?